src/app/Utils/percentFormatter.tsx

```diff
@@ -0,0 +1,4 @@
+export const percentFormatter = (value: number) => {
+    const fixedNumber = value
+    return value > 0 ? <div>&#x25B4; {fixedNumber.toFixed(2)}%</div> : <div>&#x25BE; {Math.abs(fixedNumber).toFixed(2)}%</div>;
+  }
```

Comment:
Please make sure you separate concerns by doing one thing at a time. This will improve significantly your code readability.

Also, try not to mix utility functions with HTML tags. Keep the tags in the main component.

Here's an improved version of your function:
`
export const percentFormatter = (value: number) : string => {
  const formattedValue = Math.abs(value).toFixed(2) + '%';
  const arrowSymbol = value > 0 ? '&#x25B4;' : '&#x25BE;'

  return `${arrowSymbol} ${formattedValue}`;
};
`